Apps/Clouseau/src/objects/number.lisp

```diff
@@ -48,19 +78,66 @@
              (maybe-special-float sb-ext:float-nan-p          "nan")
              (maybe-special-float sb-ext:float-trapping-nan-p "trapping-nan"))))
 
+(defmethod inspect-object-using-state :after ((object number)
+                                              (state  inspected-object)
+                                              (style  (eql :badges))
+                                              (stream t))
+  (macrolet
+      ((limits ()
+         (labels ((constant (constant-name)
+                    `((= object ,constant-name)
+                      (write-char #\Space stream)
+                      (badge stream ,(string-downcase
+                                      (symbol-name constant-name)))))
+                  (type (type)
+                    (let ((lower (let ((*package* (find-package "CL")))
+                                   (symbolicate '#:most-negative- type)))
+                          (upper (let ((*package* (find-package "CL")))
+                                   (symbolicate '#:most-positive- type))))
+                      `(,(constant lower)
+                        ,(constant upper)))))
+           `(cond
+              ,@(mappend #'type '(fixnum
+                                  single-float
+                                  double-float))
+              ,(constant 'pi)))))
+    (limits)))
+
```

Comment:
A matter of taste, but for integers, the `(type fixnum 0 4611686018427387903)` could maybe benefit from being shortened (or labelled with a badge), like `(type fixnum 0 most-positive-fixnum)`